simple-shell.nix

```diff
@@ -15,12 +15,23 @@ let
     sha256 = "1qz1lwk0j2r6jzpc5ssrlfcf1193ay5l23kwyda5wld5sm170ffw";
   }) { pkgs = np; }).mkGhc;
 
+  # This archive will be gone on 20th May. Please update when a new build of the
+  # `wip/ghc-debug` branch becomes available (which will hold for another month).
```

Comment:
I clicked the "keep" button so hopefully it doesn't disappear.

https://gitlab.haskell.org/ghc/ghc/-/jobs/312946